packages/pigeon/platform_tests/shared_test_plugin_code/lib/test_types.dart

```diff
@@ -54,12 +54,6 @@ void compareAllNullableTypesWithoutRecursion(
       true);
   expect(listEquals(allNullableTypesOne.enumList, allNullableTypesTwo.enumList),
       true);
-  // TODO(stuartmorgan): Enable this once the Dart types are fixed; see
-  // https://github.com/flutter/flutter/issues/116117
-  //for (int i = 0; i < allNullableTypesOne.listList!.length; i++) {
-  //  expect(listEquals(allNullableTypesOne.listList![i], allNullableTypesTwo.listList![i]),
-  //      true);
-  //}
```

Comment:
\o/

Reply:
It's still not fixed, but the next pr will re-add a version of this. Then I'll add a more robust test in the final sequel.

Reply:
🤦🏻 Somehow I mis-parsed this as enabling the test rather than removing it.

> It's still not fixed, but the next pr will re-add a version of this.

👍🏻